components/x-live-blog-post/src/LiveBlogPost.scss

```diff
@@ -74,3 +70,22 @@
 	border-radius: 50%;
 	background-color: oColorsByName('crimson');
 }
+
+.live-blog-post__bottom-controls {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+	margin-top: oSpacingByName('s6');
+}
+
+.live-blog-post__bottom-controls .back-to-top {
+	@include oTypographySans($scale: 1);
+	color: oColorsByName('teal');
+	text-decoration: underline;
+	margin-left: auto;
+}
+
+.live-blog-post__bottom-controls .back-to-top:hover {
+	cursor: pointer;
```

Comment:
**Question:** Do you know why this is needed? shouldn't this be default behaviour for an anchor?

Reply:
In this case it didn't work that way, so I had to add it in.

Reply:
Not sure why that happened.

Reply:
I am interested to try and understand what the problem is here rather than just apply the styling to fix it. Maybe we could look more into this before we merge

Reply:
Sure, I'll do some investigation. I do think it may have to do with this line `@include oTypographySans($scale: 1);` added to the styling.

Reply:
Running this locally with storybook in chrome and firefox its works fine for me without these styles

Reply:
@GlynnPhillips  This has to do with the browser behaviour and the design choices I made here. When an `a` tag is missing the `href` attribute, the browser takes away the `cursor: pointer` `:hover` state and renders it like a regular text. So for situations when `backToTop` is a string, it should behave like a regular `a` tag but when it is a function it wouldn't. So this override is required ensure consistent behaviour.

Reply:
Thanks @noelenwenede - that makes sense but unfortunately made me aware of this I hadn't noticed it before. I wonder if we should switch it to be a button in the instance of the prop being a function because I guess its no longer an anchor. What do you think? its extra complexity

Reply:
Yes that makes sense for accessibility. 🙌🏽